server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfile.java

```diff
@@ -38,34 +39,43 @@ public MemberProfile() {
     @Column(name="uuid")
     @AutoPopulated
     @TypeDef(type=DataType.STRING)
+    @Schema(description = "id of the member profile this entry is associated with", required = true)
     private UUID uuid;
 
     @NotNull
     @Column(name = "name", nullable = false)
+    @Schema(description = "full name of the employee", required = true)
     private String name;
 
     @Column(name="role")
+    @Schema(description = "employee's role at the company", required = true)
```

Comment:
This might be a strange question but @mkimberlin will everyone with access to this site be an employee of OCI?

Reply:
This is not a strange question. Initially...yes...everyone will be an employee of OCI.

Eventually, this may be something that is worth making multi-tenant...but that wouldn't effect this